hw/ip/rom_ctrl/doc/checklist.md

```diff
@@ -174,25 +174,25 @@ Review        | [V2_CHECKLIST_SCOPED][]               | Done        |
 
  Type         | Item                                    | Resolution  | Note/Collaterals
 --------------|-----------------------------------------|-------------|------------------
-Documentation | [DESIGN_DELTAS_CAPTURED_V2][]           | Not Started |
-Documentation | [DV_DOC_COMPLETED][]                    | Not Started |
-Testbench     | [FUNCTIONAL_COVERAGE_IMPLEMENTED][]     | Not Started |
-Testbench     | [ALL_INTERFACES_EXERCISED][]            | Not Started |
-Testbench     | [ALL_ASSERTION_CHECKS_ADDED][]          | Not Started |
-Testbench     | [SIM_TB_ENV_COMPLETED][]                | Not Started |
-Tests         | [SIM_ALL_TESTS_PASSING][]               | Not Started |
-Tests         | [FPV_ALL_ASSERTIONS_WRITTEN][]          | Not Started |
-Tests         | [FPV_ALL_ASSUMPTIONS_REVIEWED][]        | Not Started |
-Tests         | [SIM_FW_SIMULATED][]                    | Not Started |
-Regression    | [SIM_NIGHTLY_REGRESSION_V2][]           | Not Started |
-Coverage      | [SIM_CODE_COVERAGE_V2][]                | Not Started |
-Coverage      | [SIM_FUNCTIONAL_COVERAGE_V2][]          | Not Started |
-Coverage      | [FPV_CODE_COVERAGE_V2][]                | Not Started |
-Coverage      | [FPV_COI_COVERAGE_V2][]                 | Not Started |
-Code Quality  | [TB_LINT_PASS][]                        | Not Started |
-Integration   | [PRE_VERIFIED_SUB_MODULES_V2][]         | Not Started |
-Issues        | [NO_HIGH_PRIORITY_ISSUES_PENDING][]     | Not Started |
-Issues        | [ALL_LOW_PRIORITY_ISSUES_ROOT_CAUSED][] | Not Started |
+Documentation | [DESIGN_DELTAS_CAPTURED_V2][]           |    Done     |
+Documentation | [DV_DOC_COMPLETED][]                    |    Done     |
+Testbench     | [FUNCTIONAL_COVERAGE_IMPLEMENTED][]     |    Done     |
+Testbench     | [ALL_INTERFACES_EXERCISED][]            |    Done     |
+Testbench     | [ALL_ASSERTION_CHECKS_ADDED][]          |    Done     |
+Testbench     | [SIM_TB_ENV_COMPLETED][]                |    Done     |
+Tests         | [SIM_ALL_TESTS_PASSING][]               |    Done     |
+Tests         | [FPV_ALL_ASSERTIONS_WRITTEN][]          |    N/A      |
+Tests         | [FPV_ALL_ASSUMPTIONS_REVIEWED][]        |    N/A      |
+Tests         | [SIM_FW_SIMULATED][]                    |    N/A      |
+Regression    | [SIM_NIGHTLY_REGRESSION_V2][]           |    Done     |
+Coverage      | [SIM_CODE_COVERAGE_V2][]                |    Done     |
```

Comment:
The ROM checker FSM seems low on coverage, which is bringing your coverage numbers down:
https://reports.opentitan.org/hw/ip/rom_ctrl/dv/latest/cov_report/mod33.html#FSM

The checklist item does not seem to have been satisfied.

Reply:
That's true. @sriyerg: we had a bit of a discussion about this in the V2 meeting (notes [here](https://docs.google.com/document/d/164mFvT4qt6lMp2X-bnzFKUfoiGO6bYFFAriQttES3xw)). We've got an action item for Cindy to write an exclusion file which should fix things.

@prajwalaputtappa: Maybe it's easiest to wait a couple of days until everything in https://github.com/lowRISC/opentitan/issues/10580 is done, then do the sign-off afterwards?